### PR TITLE
Don't give the selected stock a background in compact search

### PR DESCRIPTION
**Pull request by WPFerg**

### CODE REVIEW

File: src/sidebars/search/search-compact.less [hunk 1]
```diff
@@ -24,6 +24,14 @@
         }
       }
     }
+
+    &.search {
+      .search-result {
+        &.selected.dark {
+          background-color: @light-background;
```
Comment (owennw) [inline src/sidebars/search/search-compact.less]: Why not a single line? `&.search .search-result.selected.dark`?

Comment (WPFerg) [inline src/sidebars/search/search-compact.less]: Good point. I've pushed up a new commit that fixes something, so I don't think it applies now.

File: src/sidebars/search/search-compact.less [hunk 2]
```diff
@@ -24,6 +24,18 @@
         }
       }
     }
+
+    &.search {
+      .search-result {
+        &.selected.dark {
```
Comment (owennw) [inline src/sidebars/search/search-compact.less]: Same comment! `&.search .search-result.selected.dark` can still be the top line here.

Comment (WPFerg) [inline src/sidebars/search/search-compact.less]: Whoops, done

**Comment by bjedrzejewski** [comment]
@WPFerg which issue is this related to?

**Comment by WPFerg** [comment]
#634

**Pull request merged**